website/docs/r/synapse_spark_pool.html.markdown

```diff
@@ -123,7 +123,7 @@ The following arguments are supported:
 
 * `spark_events_folder` - (Optional) The Spark events folder. Defaults to `/events`.
 
-* `spark_version` - (Optional) The Apache Spark version. Possible values are `2.4` , `3.1` , `3.2` and `3.3`. Defaults to `2.4`.
+* `spark_version` - (Optional) The Apache Spark version. Possible values are `2.4` , `3.1` , `3.2`, `3.3`, and `3.4`. `2.4` and `3.1` have been deprecated.  On 8 July 2024, extended support for Apache Spark Pool `3.2` will end.  Defaults to latest: `3.4`.
```

Comment:
We also shouldn't put deprecation messages with dates in the docs as the dates tend to be flexible until they are finally deprecated. Just adding the new value will suffice. 

```suggestion
* `spark_version` - (Optional) The Apache Spark version. Possible values are `2.4` , `3.1` , `3.2`, `3.3`, and `3.4`. Defaults to `2.4`.
```

Reply:
Fair enough.